HISTORY.rst

```diff
@@ -182,3 +182,24 @@ Release History
 - Adds support for inactivating skills programmatically
 - Makes decision maker run in separate thread
 - Multiple additional minor fixes and changes
+
+0.2.0 (2020-02-07)
+-------------------
+
+- Skills can now programmatically register behaviours
+- Tasks are no longer a core component of the skill, the functor pattern is used
+- Refactors the task manager
+- Adds nonces to transaction data so transactions can be verified
+- Adds documentation for the protocol generator
+- Fixes several compatibility issues between CLI and registry
+- Adds skills to connect a thermometer to an AEA
```

Comment:
Minor one, I think that generic-skills are more interesting than the thermometer :D

Reply:
will add